packages/webapp/src/components/PrivateRoute.tsx

```diff
@@ -66,11 +66,11 @@ export const PrivateRoute: React.FC = () => {
     }, [meta, loadingMeta, env, error, setEnv]);
 
     useEffect(() => {
-        if (user && environmentAndAccount) {
+        if (user && environmentAndAccount && meta && !meta.debugMode) {
```

Comment:
if `meta` is undefined, we will not enter the if block, which is a change of behavior. Is that ok?

Reply:
meta was already mandatory for user, it's just ts pleasing